client/app/components/queries/editor-components/databricks/DatabricksSchemaBrowser.jsx

```diff
@@ -84,6 +84,7 @@ export default function DatabricksSchemaBrowser({
         <Input
           className={isDatabaseSelectOpen ? "database-select-open" : ""}
           placeholder="Filter tables & columns..."
+          aria-label="Schema"
```

Comment:
```suggestion
          aria-label="Search schema"
```
...or whatever fits better (on your opinion). IMHO "Schema" is too generic and doesn't reflect the purpose of the input

Reply:
SGTM!